src/graph/graph.c

```diff
@@ -411,7 +335,7 @@ void Graph_DeleteNodes(Graph *g, NodeID *IDs, size_t IDCount) {
             // Clear, dest.
             if(destExists) {
                 if(!srcExists || (srcExists && r.delete) ) {
-                    GrB_Col_assign(M, NULL, NULL, zero, GrB_ALL, g->node_cap, r.replacementID, NULL);
+                    GrB_Col_assign(M, NULL, NULL, zero, GrB_ALL, Graph_NodeCount(g), r.replacementID, NULL);
```

Comment:
I addressed this in #99, but `r.nodeID` (destination) should be updated instead of `replacementID` (source).